ReScript.gitignore

```diff
@@ -0,0 +1,4 @@
+.DS_Store
```

Comment:
Thank you for contributing, but we don't accept OS specific rules in template files unless they're added to the OS specific template in [Global](https://github.com/github/gitignore/blob/main/Global/README.md).

See [macOS.gitignore](https://github.com/github/gitignore/blob/7b22f8ab6c85b4ef1469d72a8ba96462e2a44853/Global/macOS.gitignore) for an example.

Would you consider dropping `.DS_Store`?

Reply:
Done!